.github/labeler.yml

```diff
@@ -0,0 +1,92 @@
+documentation:
+  - 'docs/**/*'
+  - any: [ '*.rst', '!CHANGES.rst' ]
+  - '*.md'
+  - '.readthedocs.yml'
+  - 'LICENSE'
+
+dependencies:
+  - 'pyproject.toml'
+  - 'setup.*'
+  - 'requirements-*.txt'
+  - 'MANIFEST.in'
+
+# --------------------------------------- testing ---------------------------------------
+
+automation:
+  - '.github/**'
+  - '.bandit.yaml'
+  - '.codecov.yml'
+  - 'Jenkinsfile*'
+
+testing:
+  - '**/tests/**'
+  - '.github/workflows/*ci*.yml'
+  - 'conftest.py'
+  - 'tox.ini'
+
+regression_testing:
+  - '**/regtest/**'
+  - 'Jenkinsfile*'
+
+# --------------------------------------- modules ---------------------------------------
+
+assign_wcs:
+  - '**/*assign_wcs*'
+  - '**/*assign_wcs*/**'
+
+associations:
+  - '**/*association*'
+  - '**/*association*/**'
+
+Dark Current:
+  - '**/*dark_current*'
+  - '**/*dark_current*/**'
+
+dq_init:
+  - '**/*dq_init*'
+  - '**/*dq_init*/**'
+
+flatfield:
+  - '**/*flatfield*'
+  - '**/*flatfield*/**'
+
+jump:
+  - '**/*jump*'
+  - '**/*jump*/**'
+
+linearity:
+  - '**/*linearity*'
+  - '**/*linearity*/**'
+
+photom:
+  - '**/*photom*'
+  - '**/*photom*/**'
+
+ramp_fitting:
+  - '**/*ramp_fitting*'
+  - '**/*ramp_fitting*/**'
+
+saturation:
+  - '**/*saturation*'
+  - '**/*saturation*/**'
+
+# --------------------------------------- pipelines ---------------------------------------
+
+stpipe:
+  - '**/*stpipe*'
+  - '**/*stpipe*/**'
+
+pipeline:
+  - '**/*pipeline*'
+  - '**/*pipeline*/**'
+
+# --------------------------------------- instruments ---------------------------------------
+
+Wide Field Instrument (WFI):
+  - '**/*wfi*'
+  - '**/*wfi*/**'
+
+Coronagraph:
+  - '**/*coron*'
+  - '**/*coron*/**'
```

Comment:
You can remove the `Coronagraph` section. We are not going to have code for it in romancal.

Otherwise looks good!